http/static.go

```diff
@@ -77,7 +77,7 @@ func handleWithStaticData(w http.ResponseWriter, _ *http.Request, d *data, fSys
 		return http.StatusInternalServerError, err
 	}
 
-	data["Json"] = string(b)
+	data["Json"] = strings.ReplaceAll(string(b), `'`, `\'`)
```

Comment:
Encoded json is already escaped. Why this is needed?

Reply:
It escapes double quotes but not single quotes. Since single quotes is used as delimiter in the index template, it needs to be escaped manually.

https://github.com/filebrowser/filebrowser/blob/25c87883908babde073390a2e2320a8e5880a87c/frontend/public/index.html#L33